source/draw/gpu/opengl/backend/gles3/external/gles3.ooc

```diff
@@ -6,7 +6,6 @@
  * of the MIT license.  See the LICENSE file for details.
  */
 
-version(!gpuOff) {
```

Comment:
Not if the "dependent" parts are wrapped inside version flags. I will reinstate those flags